From 465fdd8d72e7f8517d67ec7b5f227f5482928ec1 Mon Sep 17 00:00:00 2001
From: "Brian C. Lane" <bcl@redhat.com>
Date: Mon, 31 Jul 2017 15:20:45 -0700
Subject: [PATCH] atari.c: Drop xlocale.h (#1476934)

glibc no longer includes this header. locale.h should work just fine.
---
 libparted/labels/atari.c | 1 -
 1 file changed, 1 deletion(-)

diff --git a/libparted/labels/atari.c b/libparted/labels/atari.c
index 36e68b9..ea4c87f 100644
--- a/libparted/labels/atari.c
+++ b/libparted/labels/atari.c
@@ -37,7 +37,6 @@
 #include <parted/endian.h>
 #include <string.h>
 #include <locale.h>
-#include <xlocale.h>
 #include <stdint.h>
 #include <ctype.h>
 #include <stddef.h>
-- 
2.9.4

